meseq.py

```diff
@@ -140,11 +140,12 @@ def box(self, x, y, text):
         self.cr.stroke()
 
         # the text
-        self.text(x, y, text)
+        self.text(x, y, text, white_bg=False)
```

Comment:
Could you please add a new example to illustrate this feature? With the existing examples, I do not see the benefit, and on the contrary some arrows are slightly damaged.